iommu_data_structures.adoc

```diff
@@ -512,15 +512,15 @@ a S-stage page table or VS-stage page table respectively.
 ], config:{lanes: 2, hspace: 1024, fontsize: 16}}
 ....
 
-A valid (`V`=1) leaf PDT entry holds the PPN of the root page of a first-stage
+A valid (`V==1`) leaf PDT entry holds the PPN of the root page of a first-stage
 page table and the `MODE` used to determine the first-stage address translation
-scheme. The `MODE` field encodings are as defined for the `MODE` field in
+scheme. The `MODE` field encodings are as defined for the `MODE` field in the
 `satp`/`vsatp` CSR.
 
 The software assigned process soft-context ID (`PSCID`) is used as the address
 space ID of the process identified by the first-stage page table.
 
-When two-stage address translation is active (`iohgatp.MODE != Bare`), the PPN
```

Comment:
I feel it renders better and aligned with all other text to have `iohgatp.MODE != Bare`.

Reply:
I was trying to align with the beginning of the subsection (section 2.2.3/subsection `fsc`) where there was no dot between `iohgatp` and `MODE` and where it was written "when `iohgatp` `MODE` is not `Bare`". But otherwise you are right that probably to keep the alignment with all other text (previous sections/subsections) it should be kept as it was (with the dot and the `!=`).